### DISCUSSION TITLE
will not register with all port forwards in place

### DISCUSSION THREAD

**Discussion by khenry72000** [GerryDazoo/Slinger#351] [General]
Slinger Remote Access Test failure ('xx.xx.xx.251', 8080) [WinError 10060] A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond
No working external portmaps. Not registering slingboxes

have 8080 going to windows10 server
have M1 5301 tried both sling box and server
using windows exe file
open windows security ports public and private
everything loads properly 

Any suggestions? Thanks in advance

**Comment by GerryDazoo** [comment]
If you don't need remote access to your server comment out you finderID.
If you do then you've still got a error in your port maps or firewall settings.
Basically the server is tying to send a message to itself though the router and it's failing.

**Comment by floormanager1** [comment]
I have something similar now I am blocking Slingbox access to the internet.  I cannot run Slinger using wifi.  If connected through an ethernet cable, the Slinger connects! [https://github.com/GerryDazoo/Slinger/discussions/304](url)

**Comment by GerryDazoo** [comment]
That doesn't make any sense to me. Is this windows? Please supply the output of ipconfig /all  when using wifi and when using Ethernet. And a copy of config.ini,

**Comment by GerryDazoo** [comment]
Khenry: Have a look at your WAN IP address that's been assigned by your ISP. If it's not a routable address you may be the victim of the dreaded double NAT that stops you from receiving any traffic from the internet.
192.168.x.x
10.x.x.x
172.16-31.x.x
Are non-routable addresses.

**Comment by floormanager1** [comment]
> ,
[SLINGBOX]
sbtype="350"
password=nxxxxxxxx
; uncomment the next lines ipaddress and port with your local network info
; if you don't want the server to automatically find your 
; slingbox on the local network. Needed if server and slingbox are
; not on the same LAN segment. ALso removes requirement for the netifaces module  
ipaddress=192.168.1.12
port=5201
; Repace with your finderid
; finderid=F295CFD982844A04220F7DD45E09671B

name=Slingbox 350

; Following are the default values
; Default resolution when server starts. You can change this and reconnect without having to restart the server
valid range = 1..15
Resolution=15

;Valid Values 1, 6, 10, 15, 20, 30, 60   depending on resolution
FrameRate=30

;Min 50 Max 8000
VideoBandwidth=3000

;Min 0 (auto) max 63
VideoSmoothness=63

; Send Iframe every n seconds Max 30
IframeRate=5

; Audio Bit Rate. Valid Options 16, 20, 32, 40, 48, 64, 96 Default 32
AudioBitRate=32

; Video Source  0, 1, 2, 3 depending on your hardware corrosponds to one of
;Composite, Component, HDMI or Tuner.
;ProHD 0=Tuner, ???
;Solo, M1 0=composite 1=Svideo 2=component
;500  0=composite, 1=component, 2=HDMI
; If you don't specify it will default to the last selected input
; So if you only have one video source connected you don't need to 
; mess with this. This number "might" be the same as the [REMOTE] code= entry but I can't say for sure. Please ket me know if you find a counter example
VideoSource = 1

[SERVER]
; local port number for the server to listen on for connections
port=63000
maxstreams=3
enableremote=yes

[REMOTE]
; see release notes for changing this if your remote doesn't work
code=2
; path to an external remote control definitions file, if the defaults
; aren't working for you or you want to change the format of the web page.
; use the supplied remote.txt as a starting point.
;include=remote.txt 
include=skyhdRemote.html

**### On cable**

D:\Slingbox 350\Slingbox>(
REM if code is not on your C drive switch to the correct drive letter
 REM g:
 REM use the full pathname to where your code is located
 cd slingbox
 slingbox_server.exe
 REM or if your using python use the full path to whatever python interpreter you're using.
 REM "c:\Program Files\Python39\python.exe" slingbox_server.py
)
The system cannot find the path specified.
Version : 3.08c Running on Windows-10-10.0.22621-SP0 pid= 12500
Using config file config.ini
Connection Manager Running on port 63000 with 3 max streams using URL slingbox.
starting up on  port 63000 8388608
Streamer Running:  3 config.ini SLINGBOX 63000 63000 8388608
Checking for slingbox at ('192.168.1.12', 5201)
('192.168.1.12', 5201) OK
Slingbox 350 Using slingbox at  ('192.168.1.12', 5201)
Streamer:  Slingbox 350 Waiting for first stream, flushing any IR requests that arrive while not connected to slingbox
Reading Custom Remote definition from skyhdRemote.html
Error reading remote definition file, using defaults
 * Serving Flask app 'slingbox_server' (lazy loading)
 * Environment: production
   WARNING: This is a development server. Do not use it in a production deployment.
   Use a production WSGI server instead.
* Debug mode: off

**### Same set-up on wifi**
D:\Slingbox 350\Slingbox>(
REM if code is not on your C drive switch to the correct drive letter
 REM g:
 REM use the full pathname to where your code is located
 cd slingbox
 slingbox_server.exe
 REM or if your using python use the full path to whatever python interpreter you're using.
 REM "c:\Program Files\Python39\python.exe" slingbox_server.py
)
The system cannot find the path specified.
Version : 3.08c Running on Windows-10-10.0.22621-SP0 pid= 12336
Using config file config.ini
Connection Manager Running on port 63000 with 3 max streams using URL slingbox.
Streamer Running:  3 config.ini SLINGBOX 63000 63000 8388608
starting up on  port 63000 8388608
Checking for slingbox at ('192.168.1.12', 5201)
Reading Custom Remote definition from skyhdRemote.html
Error reading remote definition file, using defaults
 * Serving Flask app 'slingbox_server' (lazy loading)
 * Environment: production
   WARNING: This is a development server. Do not use it in a production deployment.
   Use a production WSGI server instead.
 * Debug mode: off
Error connecting to  ('192.168.1.12', 5201)
Slingbox 350 No valid slingbox ip info found in config.ini
Finding Slingbox on local network. My IP Info =  fe80::d65d:71cb:2c28:6807%7
Finding Slingbox on local network. My IP Info =  192.168.1.18
Finding Slingbox on local network. My IP Info =  fe80::8ab8:d4af:4a05:1764%16
Finding Slingbox on local network. My IP Info =  fe80::be58:915f:d4ed:7db2%8
Finding Slingbox on local network. My IP Info =  fe80::2d87:b108:7432:cf75%18
Finding Slingbox on local network. My IP Info =  fe80::dc70:1187:3666:9868%9
Finding Slingbox on local network. My IP Info =  192.168.1.22
Finding Slingbox on local network. My IP Info =  ::1
Finding Slingbox on local network. My IP Info =  127.0.0.1
Can't find a slingbox on network. Please make sure it's plugged in and connected. Check config.ini
Slingbox 350 Giving up. Sorry…

Only happened after I blocked ports in the router.

**Comment by GerryDazoo** [comment]
You didn't post all of the info requested, If the slingbox was on the LAN that was being probed it should show up. By definition LAN traffic should not traverse the router so I can't see how it is possible for the router to have any impact.  Can I assume the WiFi is integrated into the router? Maybe the router really is doing something to block WiFi traffic from the LAN to your slingbox. It shouldn't. Check you settings.

**Comment by floormanager1** [comment]
> You didn't post all of the info requested, If the slingbox was on the LAN that was being probed it should show up. By definition LAN traffic should not traverse the router so I can't see how it is possible for the router to have any impact.

Sorry, Gerry, 
**### On Wifi**
Windows IP Configuration
Ethernet adapter Ethernet:

   Media State . . . . . . . . . . . : Media disconnected
   Connection-specific DNS Suffix  . :

Ethernet adapter Ethernet 2:

   Media State . . . . . . . . . . . : Media disconnected
   Connection-specific DNS Suffix  . :

Wireless LAN adapter Local Area Connection* 1:

   Media State . . . . . . . . . . . : Media disconnected
   Connection-specific DNS Suffix  . :

Wireless LAN adapter Local Area Connection* 10:

   Media State . . . . . . . . . . . : Media disconnected
   Connection-specific DNS Suffix  . :

Wireless LAN adapter WiFi:

   Connection-specific DNS Suffix  . :
   Link-local IPv6 Address . . . . . : fe80::dc70:1187:3666:9868%9
   IPv4 Address. . . . . . . . . . . : 192.168.1.22
   Subnet Mask . . . . . . . . . . . : 255.255.255.0
   Default Gateway . . . . . . . . . : 192.168.1.1

**### On cable**

Windows IP Configuration


Ethernet adapter Ethernet 2:

   Media State . . . . . . . . . . . : Media disconnected
   Connection-specific DNS Suffix  . :

Wireless LAN adapter Local Area Connection* 1:

   Media State . . . . . . . . . . . : Media disconnected
   Connection-specific DNS Suffix  . :

Wireless LAN adapter Local Area Connection* 10:

   Media State . . . . . . . . . . . : Media disconnected
   Connection-specific DNS Suffix  . :

Ethernet adapter Ethernet:

   Connection-specific DNS Suffix  . :
   Link-local IPv6 Address . . . . . : fe80::d65d:71cb:2c28:6807%7
   IPv4 Address. . . . . . . . . . . : 192.168.1.18
   Subnet Mask . . . . . . . . . . . : 255.255.255.0
   Default Gateway . . . . . . . . . : 192.168.1.1

Wireless LAN adapter WiFi:

   Media State . . . . . . . . . . . : Media disconnected
   Connection-specific DNS Suffix  . :
Is this what you wanted to see?  The Slingbox and the Slinger are on the same router.  The Slingboxes are now on a blacklist to prevent Mr. Sling and his wicked ways.  I could not work out why I could no longer connect with the Slinger Server on wifi, until I removed these blocks (while unplugged from the internet).

I do not understand it either; just reporting what I observed but as you can see from the output, the same server, connected by cable or wifi gives two differnet results (cable works, wifi does not).  It no lonfger impacts me as I have a dedicated server cable-connected.

**Comment by GerryDazoo** [comment]
Yeah, the router should not block local WiFi traffic to the LAN. See my edited earlier response.
